### PR TITLE
Update bitcoin_pt_BR.ts

### PR DESCRIPTION
**Pull request by mobtap**
I recommend renaming the bitcoin_XXXX files to doge_XXXX

### CODE REVIEW

**Comment by patricklodder** [comment]
> bitcoin_XXXX files to doge_XXXX

For 1.14 the choice was made to keep it as-is. Would be too big a change for a leaf branch that will only see minor updates. But we can check thoughts with @rnicoll for 1.21! Thoughts?

**Comment by rnicoll** [comment]
Yeah I've intentionally not renamed the files, so that when there's changes in Bitcoin Core, we can port them to Dogecoin relatively simply.

**Review by rnicoll — APPROVED** [review]
utACK - did spot checks with Google Translate, but if someone who knows Portuguese can verify it would be great.

**Comment by mobtap** [comment]
I am here for this and whatever else you need, I really want to represent Brazil in this big step! I have full interest in commitment!  @patricklodder @rnicoll

**Comment by patricklodder** [comment]
Would love it if someone that speaks pt_BR could have a look!

File: src/qt/locale/bitcoin_pt_BR.ts [hunk 1]
```diff
@@ -680,11 +680,11 @@
     </message>
     <message>
         <source>This label turns red if any recipient receives an amount smaller than the current dust threshold.</source>
-        <translation>Este texto fica vermelho se qualquer destinatário receber uma quantidade menor que que o dust.</translation>
+        <translation>Este texto fica vermelho se qualquer destinatário receber uma quantidade menor que que a poeira.</translation>
```
Comment (matheus-delazeri) [inline src/qt/locale/bitcoin_pt_BR.ts:683]: I think that only one "que" is needed in this line.

Comment (mobtap) [reply src/qt/locale/bitcoin_pt_BR.ts:683]: Thanks!

Comment (mobtap) [inline src/qt/locale/bitcoin_pt_BR.ts]: For Brazilian Portuguese, I believe that an easy interpretation would be a percentage and not dust!

File: src/qt/locale/bitcoin_pt_BR.ts [hunk 2]
```diff
@@ -680,11 +680,11 @@
     </message>
     <message>
         <source>This label turns red if any recipient receives an amount smaller than the current dust threshold.</source>
-        <translation>Este texto fica vermelho se qualquer destinatário receber uma quantidade menor que que o dust.</translation>
+        <translation>Este texto fica vermelho se qualquer destinatário receber uma quantidade menor que a poeira.</translation>
```
Comment (mobtap) [inline src/qt/locale/bitcoin_pt_BR.ts:683]: For Brazilian Portuguese, I believe that an easy interpretation would be a percentage and not dust!

**Review by patricklodder — APPROVED** [review]
:+1:

**Pull request merged**